optimum/exporters/tasks.py

```diff
@@ -392,6 +392,10 @@ class TasksManager:
             "feature-extraction",
             onnx="CLIPTextOnnxConfig",
         ),
+        "clip-vision-model": supported_tasks_mapping(
```

Comment:
Maybe you can add `clip-vision-with-projection` since you also added an ONNX config for the model with projection. Like `clip-text-with-projection` right below.